temporalio/client.py

```diff
@@ -1108,6 +1108,46 @@ async def get_worker_task_reachability(
             )
         )
 
+    async def describe_task_queue(
+        self,
+        task_queue: str,
+        task_queue_types: Sequence[TaskQueueType] = [],
+        report_pollers: bool = False,
+        report_stats: bool = False,
+        rpc_metadata: Mapping[str, str] = {},
+        rpc_timeout: Optional[timedelta] = None,
+    ) -> TaskQueueDescription:
+        """Describe task queue.
```

Comment:
nit: This comment is redundant.  Can you give a flavor for the type of information I should expect as a response and why I would use each one?

At this point I assumed I would get some info if I check neither and was wondering what it was.  Wasn't until I looked at the code that I realized it wouldn't work.

Reply:
:+1: I will clarify that this returns poller and stats information and that currently you need to provide at least one of the booleans.